app/javascript/modCenter/singleArticle/singleArticle.test.jsx

```diff
@@ -16,11 +17,28 @@ const testArticle = {
   },
 };
 
+const testArticle2 = {
+  id: 2,
+  title:
+    'An article title that is quite very actually rather extremely long with all things considered',
+  path:
+    'an-article-title-that-is-quite-very-actually-rather-extremely-long-with-all-things-considered-fi8',
+  publishedAt: '2019-06-24T09:32:10.590Z',
+  cachedTagList: '',
+  user: {
+    articles_count: 1,
+    name: 'howdy',
+  },
+};
+
+const truncatedTitle =
+  'An article title that is quite very actually rather extre...';
```

Comment:
I think it makes sense to put this in the test itself and not outside of the scope.

Reply:
ok!